azurerm/internal/services/containers/kubernetes_cluster_node_pool_resource_test.go

```diff
@@ -870,6 +870,38 @@ func testAccKubernetesClusterNodePool_sameSize(t *testing.T) {
 	})
 }
 
+func TestAccKubernetesClusterNodePool_ultraSSD(t *testing.T) {
+	checkIfShouldRunTestsIndividually(t)
+	testAccKubernetesClusterNodePool_ultraSSD(t)
+}
+
+func testAccKubernetesClusterNodePool_ultraSSD(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_kubernetes_cluster_node_pool", "test")
+	r := KubernetesClusterNodePoolResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.ultraSSD(data, false),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep(), {
+			Config: r.ultraSSD(data, false),
```

Comment:
presumably this should be true so there's a change here:

```suggestion
			Config: r.ultraSSD(data, true),
```

but since this is ForceNew this test is superfluous?